src/WrappedVault.sol

```diff
@@ -303,7 +305,7 @@ contract WrappedVault is Owned, ERC20, IWrappedVault {
         // Any unclaimed rewards can still be claimed
         rewardsPerToken.lastUpdated = start.toUint32();
 
-        emit RewardsSet(reward, block.timestamp.toUint32(), rewardsInterval.end, rate, rewardsAfterFee, protocolFeeTaken, frontendFeeTaken);
+        emit RewardsSet(reward, block.timestamp.toUint32(), rewardsInterval.end, (rate * (end - start)), rewardsAfterFee, protocolFeeTaken, frontendFeeTaken);
```

Comment:
I think you meant to replace `rewardsAfterFee` with `(rate * (end - start))` rather than `rate`?

Reply:
F